packaging/before-uninstall.sh

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env bash
+
+######### Let's support alpine installations
+PATH=${PATH}:/usr/local/bin
+
+################################################################################
+############################ GLOBAL VARIABLES ##################################
+################################################################################
+####### IMPORTANT: PHP_INSTRUMENTATION_DIR is needed otherwise fpm will fail when generating
+#######            the apk distribution with an Invalid tar stream error.
+PHP_INSTRUMENTATION_DIR=/opt/open-telemtry/php-instrumentation
+BACKUP_EXTENSION=".instrumentation.uninstall.bck"
+
+################################################################################
+########################## FUNCTION CALLS BELOW ################################
+################################################################################
+
+################################################################################
+#### Function php_command ######################################################
+function php_command() {
+    PHP_BIN=$(command -v php)
+    ${PHP_BIN} -d memory_limit=128M "$@"
+}
+
+################################################################################
+#### Function php_ini_file_path ################################################
+function php_ini_file_path() {
+    php_command -i \
+        | grep 'Configuration File (php.ini) Path =>' \
+        | sed -e 's#Configuration File (php.ini) Path =>##g' \
+        | head -n 1 \
+        | awk '{print $1}'
+}
+
+################################################################################
+#### Function php_api ##########################################################
+function php_api() {
+    php_command -i \
```

Comment:
if there was a `php-config --api`, we could get everything more effectively from there...